### PR TITLE
fix forwarded metrics

### PR DESCRIPTION
**Pull request by mlsmaycon**
### Checklist
- [x] Is it a bug fix
- [ ] Is a typo/documentation fix
- [ ] Is a feature enhancement
- [ ] It is a refactor
- [ ] Created tests that fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: signal/server/signal.go [hunk 1]
```diff
@@ -62,13 +64,15 @@ func (s *Server) Send(ctx context.Context, msg *proto.EncryptedMessage) (*proto.
...
 	if dstPeer, found := s.registry.Get(msg.RemoteKey); found {
+		start := time.Now()
 		//forward the message to the target peer
 		if err := dstPeer.Stream.Send(msg); err != nil {
 			log.Errorf("error while forwarding message from peer [%s] to peer [%s] %v", msg.Key, msg.RemoteKey, err)
 			//todo respond to the sender?
 
 			s.metrics.MessageForwardFailures.Add(ctx, 1, metric.WithAttributes(attribute.String(labelType, labelTypeError)))
 		} else {
+			s.metrics.MessageForwardLatency.Record(ctx, float64(time.Since(start).Nanoseconds())/1e6, metric.WithAttributes(attribute.String(labelType, labelTypeMessage)))
```
Comment (pappz) [inline signal/server/signal.go:75]: Instead of /1e6 we could use time.Since(start).Milliseconds(), I think.

File: signal/server/signal.go [hunk 2]
```diff
@@ -118,22 +122,21 @@ func (s *Server) ConnectStream(stream proto.SignalExchange_ConnectStreamServer)
 		} else if err != nil {
 			return err
 		}
-		start := time.Now()
 
 		log.Debugf("received a new message from peer [%s] to peer [%s]", p.Id, msg.RemoteKey)
 
 		// lookup the target peer where the message is going to
 		if dstPeer, found := s.registry.Get(msg.RemoteKey); found {
+			start := time.Now()
```
Comment (lixmal) [inline signal/server/signal.go:130]: Any reason you don't want to include the map lookup in the latency anymore? My idea was to include the whole operation

File: signal/server/signal.go (hunk 1 above)
Comment (lixmal) [reply signal/server/signal.go:75]: If you do that we'll lose precision, as `Milliseconds` returns an int, but we have fractional milliseconds in the histogram buckets: 


https://github.com/netbirdio/netbird/blob/main/signal%2Fmetrics%2Fapp.go#L90-L104

But maybe we don't need those small buckets in the first place. We should take a look at the actual data.

File: signal/server/signal.go (hunk 2 above)
Comment (mlsmaycon) [reply signal/server/signal.go:130]: I would rather keep the map lookup as a separate metric and have the latency only linked to the send operation

File: signal/server/signal.go (hunk 1 above)
Comment (mlsmaycon) [reply signal/server/signal.go:75]: let's keep them for now

**Pull request merged**